docs/resources/excel-samples.md

```diff
@@ -38,6 +38,70 @@ function main(workbook: ExcelScript.Workbook) {
 }
 ```
 
+### Read the active cell
+
+This script logs the value of the current active cell. If multiple cells are selected, the top-leftmost cell will be logged.
+
+```typescript
+function main(workbook: ExcelScript.Workbook) {
+  // Get the current active cell in the workbook.
+  let cell = workbook.getActiveCell();
+
+  // Log that cell's value.
+  console.log(`The current cell's value is ${cell.getValue()}`);
+}
+```
+
+### Change an adjacent cell
+
+This script gets adjacent cells by using relative references. Note that if the active cell is on the top row, part of the script fails, because it references the cell above the currently selected one.
```

Comment:
```suggestion
This script gets adjacent cells using relative references. Note that if the active cell is on the top row, part of the script fails, because it references the cell above the currently selected one.
```